operations/configuration_management/NCN_Node_Personalization.md

```diff
@@ -39,7 +39,8 @@ All of the following are prerequisites on the node where this procedure is being
 
 1. (`ncn-m#`) Back up the current CFS state.
```

Comment:
I will just point out that I think these instructions will need further changes for 1.4, though probably in a follow-up PR. One of the changes for Alice is to stop using separate NCN personalization and customization configs (i.e., use the same CFS config for both and ensure the specific Ansible plays do the right thing in each context). So going forward we probably won't need steps for editing the file to remove the `ncn-image-customization` config from the input file.

Reply:
Yup, I figured as much. But like you, I think that's best done in a separate PR.